src/modules/folder/folder.repository.ts

```diff
@@ -447,6 +453,25 @@ export class SequelizeFolderRepository implements FolderRepository {
     return folders.map((folder) => this.toDomain(folder));
   }
 
+  async calculateFolderSize(
+    folderUuid: string,
+    timeoutSeconds = 15,
+  ): Promise<number> {
+    const calculateSizeQuery = `SELECT calculate_folder_size('${folderUuid}') AS totalSize;`;
```

Comment:
Use Sequelize variable interpolation here to avoid SQL Injections

```
const calculateSizeQuery = 'SELECT calculate_folder_size(:folderUuid) AS totalSize';

FolderModel.sequelize.query(calculateSizeQuery, {
  replacements: { folderUuid: folderUuid },
  type: QueryTypes.SELECT
})
```